apps/meteor/app/livechat/server/api/lib/inquiries.ts

```diff
@@ -39,24 +40,25 @@ export async function findInquiries({
 	pagination: { offset: number; count: number; sort: Record<string, number> };
 }): Promise<PaginatedResult<{ inquiries: Array<ILivechatInquiryRecord> }>> {
 	const department = await applyDepartmentRestrictions(userId, filterDepartment);
-
+	const defaultSort = await LivechatInquiry.getSortingQuery();
```

Comment:
@murtaza98 , Thinking carefully about this, having this sorting call based on a setting directly in the model feels a bit weird. Models shouldn't have this dependency. Wdyt of updating `findInquiries` to receive as a parameter the sorting query? 

That also removes the dependency of this model into settings model, and allow us to use the cached settings, since we'll be using it from the outside 👀

Reply:
Makes sense. Moved the entire logic of fetching the settings out of this model